src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,67 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  query: string,
+  setQuery: (query: string) => void,
+  filter: string,
+  setFilter: (filter: string) => void,
```

Comment:
you can still use created enum `FilterValues` but it would be better to place it somewhere separately (not in App.tsx)
```suggestion
  filter: FilterValues,
  setFilter: (filter: FilterValues) => void,
```